src/app.gresource.xml

```diff
@@ -37,8 +37,11 @@
     <file>Library/demos/Custom Widget/main.ui</file>
     <file>Library/demos/Custom Widget/main.js</file>
     <file>Library/demos/Custom Widget/main.json</file>
-     <file>Library/demos/Switch/main.json</file>
+    <file>Library/demos/Switch/main.json</file>
     <file>Library/demos/Switch/main.js</file>
     <file>Library/demos/Switch/main.blp</file>
+    <file>Library/demos/Spinner/main.blp</file>
+    <file>Library/demos/Spinner/main.js</file>
+    <file>Library/demos/Spinner/main.json</file>
```

Comment:
not needed anymore - please merge `main`

https://github.com/sonnyp/Workbench/pull/200